packages/ember-views/tests/views/view/attribute_bindings_test.js

```diff
@@ -193,6 +193,22 @@ QUnit.test("should update attribute bindings with micro syntax", function() {
   ok(!view.$().prop('disabled'), "updates disabled property when false");
 });
 
+QUnit.test("should allow namespaced attributes in micro syntax", function () {
+  view = EmberView.create({
+    attributeBindings: ['xlinkHref:xlink:href'],
+    xlinkHref: '/foo.png'
+  });
+
+  run(function() {
+    view.createElement();
+  });
+  equal(view.$().attr('xlink:href'), '/foo.png', "namespaced attribute is set");
+
+  run(function () {
+    view.set('xlinkHref', '/bar.png');
+  });
+  equal(view.$().attr('xlink:href'), '/bar.png', "namespaced attribute is updated");
```

Comment:
Don't know.  Was going to try it out with an interactive saucelabs session just now, but both it and master stop with 0 of 3 assertions passed: http://imgur.com/rP5PVTA.  I can try on a VM tonight to see what's going on.